docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+.. inclusion-marker-start-do-not-remove
+
+Building a Conda environment with GPU support for Horovod
+=========================================================
+
+In this section we describe how to build Conda environments for my deep learning projects using 
```

Comment:
In this section we describe --> This section describes 
remove "my"

Reply:
Done.